### PR TITLE
Set default pipeline idx in PipelineManager constructor

### PR DESCRIPTION
**Pull request by mcm001**
Addresses https://github.com/PhotonVision/photonvision/issues/1285

### CODE REVIEW

File: photon-core/src/main/java/org/photonvision/vision/processes/PipelineManager.java [hunk 1]
```diff
@@ -70,10 +71,19 @@ public class PipelineManager {
         if (userPipelines.isEmpty()) addPipeline(PipelineType.Reflective);
 
         calibration3dPipeline = new Calibrate3dPipeline(uniqueName);
+
+        // We know that at this stage, VisionRunner hasn't yet started so we're good to do this from
```
Comment (mdurrani808) [inline photon-core/src/main/java/org/photonvision/vision/processes/PipelineManager.java:75]: Will this always be true?

**Review by mdurrani808 — APPROVED** [review]
Looks good to me, the naming is a little confusing given requested / current but logically it seems good. I'm not able to test but I believe you have, would be good to spin up some code and ensure that changing the pipeline via PhotonLib still works as intended.

File: photon-core/src/main/java/org/photonvision/vision/processes/PipelineManager.java (hunk 1 above)
Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/vision/processes/PipelineManager.java:75]: Yes absolutely true. PipelineManager is a prerequisite for VisionRunner existing

**Comment by mcm001** [comment]
Yeah I tested switching in/out of driver mode, i'll make sure to test changing pipeline index.

Would be really cool to set up integration tests

**Pull request merged**